recipes/scikit-learn/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.17.1" %}
+
+{% set variant = "openblas" %}
+
+package:
+  name: scikit-learn
+  version: {{ version }}
+
+source:
+  fn: scikit-learn-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-learn/scikit-learn-{{ version }}.tar.gz
+  md5: a2f8b877e6d99b1ed737144f5a478dfc
+
+build:
+  number: 200
+  # We lack openblas on Windows, and therefore can't build numpy there either currently.
+  skip: true  # [win]
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+#    - nosetests sklearn --exe
```

Comment:
I'm not having much luck running the tests. So, if anyone has any suggestions, please let me know.